src/loaders/agent.js

```diff
@@ -48,17 +47,18 @@ export class Agent extends AgentBase {
     // Future work is being planned to evaluate removing this behavior from the backend, but for now we must ensure this call is made
     this.desiredFeatures.add(PageViewEvent)
 
+    this.runSoftNavOverSpa = [...this.desiredFeatures].some(instr => instr.featureName === FEATURE_NAMES.softNav)
```

Comment:
i think this looks like it still needs to be hooked up to the feature_flags mechanism

Reply:
The final output for `runSoftNavOverSpa` is actually completed on the next line inside configure of configure.js, L48. The information I needed to make the decision is fragmented into different scopes, hence I had to split this up. Didn't want to change `configure` and pass desiredFeatures or a separate temp flag in (this line can be interpreted as the temp flag anyways).